test/db/analysis/vars

```diff
@@ -427,11 +427,11 @@ afvR
       arg1  0x6
       arg2  0x8
       arg3  0xc
-   var_10h  0x4,0x2c
```

Comment:
Looks like a regression?

Reply:
This is the result of https://github.com/rizinorg/rizin/pull/3591/commits/7263b0648f432766325db037d4e1f51d6b139852

During the function `var`/`arg` detection, there is a rule to check, if the `sp` or `bp` is only moved (something like `add r0, sp, 0`).
If this is the case no variable is added.

But while checking esil for variables, this rule was ignored and the variable was added anyways. Hence the `var_10h` variable was added, which is only accessed when the `sp` is moved to a `gpr`.